lib/is.js

```diff
@@ -69,6 +69,12 @@ const validValue = (value) => {
 module.exports.validValue = validValue;
 
 
+const validLabelValue = (value) => {
+    return Number.isFinite(value) || isString(value) || value === null || value === undefined;
```

Comment:
boolean is probably fine as well thought that could just be expressed as a string perhaps?

Reply:
Now supporting boolean too.